snap/snapcraft.yaml.in

```diff
@@ -25,12 +25,13 @@ parts:
       unset GOROOT
       export GOPATH=$(realpath ../go)
       cd ${GOPATH}/src/github.com/${SNAPCRAFT_PROJECT_NAME}/runtime
-      QEMU_ARCH=$(arch)
```

Comment:
Nit: This unused variable (?) removal should really be on a separate commit as it's unrelated to the version check.